tests/unit_and_functional/test_git.py

```diff
@@ -24,22 +24,22 @@
 
 from pytest import raises
 from unittest.mock import patch, MagicMock
+from conftest import on_error
 
-from bodywork.constants import SSH_DIR_NAME, SSH_GITHUB_KEY_ENV_VAR
+from bodywork.constants import SSH_DIR_NAME, SSH_PRIVATE_KEY_ENV_VAR
 from bodywork.git import (
-    ConnectionPrototcol,
+    ConnectionProtocol,
     download_project_code_from_repo,
     get_connection_protocol,
-    get_remote_repo_host,
-    GitRepoHost,
-    setup_ssh_for_github
+    setup_ssh_for_git_host,
+    get_ssh_public_key_from_domain,
 )
 
 
 def test_that_git_project_repo_can_be_cloned(
     setup_bodywork_test_project: Iterable[bool],
     project_repo_connection_string: str,
-    cloned_project_repo_location: Path
+    cloned_project_repo_location: Path,
```

Comment:
I'm not sure that comma is meant to be there.

Reply:
Good spot!

Reply:
It's Black that's adding the comma's...I can't say I agree with that.